packages/core/src/config.js

```diff
@@ -291,6 +295,7 @@ export const snapshotSchema = {
         domTransformation: { $ref: '/config/snapshot#/properties/domTransformation' },
         enableLayout: { $ref: '/config/snapshot#/properties/enableLayout' },
         sync: { $ref: '/config/snapshot#/properties/sync' },
+        multiDOM: { $ref: '/config/snapshot#/properties/multiDOM' },
```

Comment:
is `multiDOM` confirmed?

Reply:
Doesnt sound very straightforward for end user. `captureMultipleWidths` seems better - it does exactly what it says.